components/PostCard.tsx

```diff
@@ -8,6 +8,36 @@ interface Props {
   post: PostDocumentWithoutContent;
 }
 
+const DraftPost = () => {
+  return (
+    <div className="draft-post">
+      <div className="triangle-draft triangle triangle-item">
+        <span className="triangle-text-draft triangle-text">DRAFT</span>
+      </div>
+      <div className="small-triagles-draft">
+        <div className="small-triagles-item-draft1 triangle small-triagles-item-draft small-triagles-item"></div>
+        <div className="small-triagles-item-draft2 triangle small-triagles-item-draft small-triagles-item"></div>
+      </div>
+    </div>
+  );
+};
+
+const FuturePost = () => {
+  return (
+    <div className="future-post">
+      <div className="triangle-future triangle triangle-item">
+        <span className="triangle-text-future triangle-text">FUTURE</span>
+        <span className="triangle-text-future-post triangle-text"> POST</span>
+      </div>
+      <div className="small-triagles-future">
+        <div className="small-triagles-item-future1 triangle small-triagles-item-future small-triagles-item"></div>
+        <div className="small-triagles-item-future2 triangle small-triagles-item-future small-triagles-item"></div>
```

Comment:
@ntorbinskiy discussed this thing in the morning, find a better name for 1, and 2